worker/uniter/runner/jujuc/network-get.go

```diff
@@ -32,6 +35,12 @@ type NetworkGetCommand struct {
 	out cmd.Output
 }
 
+// Type alias for Domain resolution function for testing
+type resolver = func(host string) (addrs []string, err error)
+
+var lookupFunc = net.LookupHost
+var LookupHost = &lookupFunc
```

Comment:
Might it be nicer to remove the reference (and dereference on line 130), assign directly, and just patch the address in the test?
```
testing.PatchValue(&jujuc.LookupHost, lookupHost)
```
Maybe even keep it private and export it for testing only...

Reply:
Since testing is in different package this would have to be public. Options are to stay public; make a different testing package for the one test case; make an testing export package which exports only those symbols that are needed for testing (as done else where); or merge the testing and code package, if we really want the symbol to stay private. I favor just leaving the symbol public since the code really should go away soon anyway.